src/pretalx/schedule/exporters.py

```diff
@@ -179,15 +195,15 @@ def get_data(self, **kwargs):
                         "rooms": {
                             str(room["name"]): [
                                 {
+                                    "url": talk.submission.urls.public.full(),
                                     "id": talk.submission.id,
                                     "guid": talk.uuid,
-                                    "logo": talk.submission.urls.image,
                                     "date": talk.local_start.isoformat(),
                                     "start": talk.local_start.strftime("%H:%M"),
+                                    "logo": talk.submission.image_url or None,
```

Comment:
This will produce a relative URL, is that really what you want?

I suspect the correct solution is something like `talk.submission.urls.image.full() if talk.submission.image else None`.